src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
```

Comment:
'shape' is defined in the Figure interface and the 'Triangle' class already implements it.